_posts/2022-11-25-kit-os-2023.1.0.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Kit OS Update 2023.1.0
+---
+
+Our Kit team has been hard at work and are delighted to announce the release of [Student Robotics OS 2023.1.0](https://studentrobotics.org/docs/kit/brain_board/updates) 🎉. The full changelog for this release is on the [updates page](https://studentrobotics.org/docs/kit/brain_board/updates) but we wanted to highlight a few of the notable improvements!
+
+## Remote Debugging
+
+If there’s one piece of advice we give to our teams it’s “Test, Test, Test” and with [Remote Debugging](https://studentrobotics.org/docs/programming/editors/vscode#remote-debugging) we hope to make this even easier!
+
+Using [VS Code](https://code.visualstudio.com/) (or an editor that supports [DAP](https://microsoft.github.io/debug-adapter-protocol/)) competitors can now step through their code while it is executing on the robot. This will allow competitors to inspect variables, change execution flow, and gain a much deeper understanding of what their robot is doing at that very moment.
```

Comment:
```suggestion
Using [VS Code](https://code.visualstudio.com/) (or any editor that supports [DAP](https://microsoft.github.io/debug-adapter-protocol/)) competitors can now step through their code while it is executing on the robot. This will allow competitors to inspect variables, change execution flow, and gain a much deeper understanding of what their robot is doing at that very moment.
```